lib/elements/rebar_hook.rb

```diff
@@ -0,0 +1,48 @@
+require './db/base'
+require './lib/materials/steel'
+
+module Elements
+  class RebarHook < Materials::Steel
```

Comment:
again, this relation between `RebarHook` and `Materials::Steel`would make more sense if a rebar hook has a `material` attribute which points to a steel material instance

Reply:
Yeah, In this better POV proposed, RebarHook and Rebar are classes of the namespace Elements::RebarHook

The Base class of this namespace has the constructor